resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/BucketRemover.scala

```diff
@@ -32,7 +32,17 @@ object BucketRemover {
         implicit ev: ApplicativeAsk[F, TraceId]
       ): F[Option[BucketToRemove]] =
         a.bucket
-          .traverse(b => deps.storageService.deleteBucket(a.googleProject, b, true).compile.drain.as(a))
+          .flatTraverse { b =>
+            deps.storageService.deleteBucket(a.googleProject, b, true).compile.last.map { resultOpt =>
```

Comment:
how come we don't need `.drain` anymore?

Reply:
`.drain` is saying "we don't care about the return type of `deleteBucket`, but here we do. Depending on whether it's `true` or `false`, we're doing things a bit differently